deployment/testing-v4/terraform/modules/bucket-permissions/main.tf

```diff
@@ -30,13 +30,13 @@ variable "service_account_ingest" {
 
 resource "google_storage_bucket_iam_binding" "private" {
   bucket  = var.bucket_private
-  role    = "roles/storage.objectAdmin"
+  role    = "roles/storage.admin"
```

Comment:
The GCS gateway requires `storage.admin` permissions as per https://docs.min.io/docs/minio-gateway-for-gcs.html. Using just `objectAdmin` leads to weird permission errors when copying files into buckets.